sites/bizbash/server/styles/index.scss

```diff
@@ -19,8 +19,27 @@ $black:     #000;
 $white:     #fff;
 $gray-600:  #4d4d4d;
 
+// Map
+$theme-site-bizbash-map: (
+  dark: (
+    default: $white,
+    hover: $white,
+    active: $white
+  ),
+  light: (
+    default: $primary,
+    hover: $white,
+    active: $white
+  ),
+  lighter: (
```

Comment:
This doesn't follow the stand convention for this variable... if anything, the root var should support merging additional values from an extending site

Reply:
This was a stopping point for last night, hence the addition of a "ToDo" in my initial merge comment.